src/main/groovy/eu/plumbr/integration/plugin/PlumbrIntegrationPlugin.groovy

```diff
@@ -0,0 +1,18 @@
+package eu.plumbr.integration.plugin
+
+import eu.plumbr.integration.artifactory.ArtifactoryIntegration
+import eu.plumbr.integration.jira.JiraIntegration
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class PlumbrIntegrationPlugin implements Plugin<Project> {
```

Comment:
Not really an appropriate name, I think. We do not integrate here with Plumbr itself, do we?